Buttplug.Server.Managers.ErosTek/ET312Device.cs

```diff
@@ -0,0 +1,398 @@
+﻿using System;
+using System.IO.Ports;
+using System.Text;
+using System.Threading.Tasks;
+using System.Timers;
+using Buttplug.Core;
+using Buttplug.Core.Messages;
+
+namespace Buttplug.Server.Managers.ETSerialManager
+{
+    public class ET312Device : ButtplugDevice
+    {
+        private SerialPort serialPort;
+        private byte boxkey;
+        private double speed;
+        private double position;
+        private double currentPosition;
+        private double increment;
+        private double fade;
+        private double updateInterval;
+        private Timer updateTimer;
+        private object movementLock;
```

Comment:
prefix private fields with `_` (underscore) so that when referenced in a method with a matching argument name, you're not needed to type `this.position` instead it's just `_position`

Reply:
You can use Visual Studio's `Ctrl` + `R`, `Ctrl` + `R` (yeah, pressed twice) to rename all instances in visual studio